### PR TITLE
Revert 61cdae0ce58bcbe048b143356fd9ded821225657 to fix CI

### PR DESCRIPTION
**Pull request by huydhn**
https://github.com/pytorch/pytorch/commit/61cdae0ce58bcbe048b143356fd9ded821225657

### CODE REVIEW

File: test/test_modules.py [hunk 1]
```diff
@@ -490,7 +489,6 @@ def test_gradgrad(self, device, dtype, module_info, training):
         self._test_gradients_helper(device, dtype, module_info, training, gradgradcheck)
 
     @onlyCUDA
-    @with_tf32_off  # Turn off TF32 to compute at full precision https://github.com/pytorch/pytorch/issues/86798
```
Comment (malfet) [inline test/test_modules.py:493]: This should be fine to leave, shouldn't it?

Comment (huydhn) [reply test/test_modules.py:493]: Yeah, this should be ok to stay

**Review by malfet — APPROVED** [review]

**Comment by huydhn** [comment]
Will merge once lint passes as Windows CI is already broken

**Pull request closed without merging**